diagnostic.md

```diff
@@ -38,7 +40,9 @@ $base-margin: 1.5em;
 ```
 
 ```scss
-// your response here
+li {
+  margin: $base-margin * .75;
```

Comment:
Correct. but math is wrong. You want to **halve** the `$base-margin` so it would need to be `margin: $base-margin * .5;`